### PR TITLE
Remove gogoproto stdtime and stdduration options

### PR DESCRIPTION
**Pull request by htdvisser**
#### Summary

This pull request removes a number of `gogoproto.stdtime` and `gogoproto.stdduration` options.

To avoid conflicts, it does not touch the proto files that other team members are working on (removing embed and nullable from end_device, datewayserver, join, lorawan, messages, metadata, regional).

Refs #2798

#### Changes

- Introduce helpers in `pkg/ttnpb/stdtime.go` and `pkg/ttnpb/stdduration.go`
- Remove options
- Use those helpers where needed
- Allow CLI to set those fields

#### Testing

This should all be covered by existing tests.

##### Regressions

nil

#### Notes for Reviewers

nil

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: pkg/events/grpc/grpc.go [hunk 1]
```diff
@@ -166,11 +166,10 @@ func (srv *EventsServer) Stream(req *ttnpb.StreamEventsRequest, stream ttnpb.Eve
 	if err != nil {
 		return err
 	}
-	now := time.Now().UTC()
 	startEvent := &ttnpb.Event{
 		UniqueId:       events.NewCorrelationID(),
 		Name:           "events.stream.start",
-		Time:           &now,
+		Time:           ttnpb.ProtoTimePtr(time.Now().UTC()),
```
Comment (johanstokking) [inline pkg/events/grpc/grpc.go]: Here `.UTC()` is preserved

File: pkg/applicationserver/io/packages/redis/registry.go [hunk 2]
```diff
@@ -230,8 +230,7 @@ func (r ApplicationPackagesRegistry) SetAssociation(ctx context.Context, ids *tt
 				pb = &ttnpb.ApplicationPackageAssociation{}
 			}
 
-			now := time.Now().UTC()
-			pb.UpdatedAt = &now
+			pb.UpdatedAt = ttnpb.ProtoTimePtr(time.Now())
```
Comment (johanstokking) [inline pkg/applicationserver/io/packages/redis/registry.go:233]: Here is isn't. Any reason for that?

File: pkg/applicationserver/io/packages/redis/registry.go [hunk 3]
```diff
@@ -426,8 +425,7 @@ func (r ApplicationPackagesRegistry) SetDefaultAssociation(ctx context.Context,
 				pb = &ttnpb.ApplicationPackageDefaultAssociation{}
 			}
 
-			now := time.Now().UTC()
-			pb.UpdatedAt = &now
+			pb.UpdatedAt = ttnpb.ProtoTimePtr(time.Now())
```
Comment (adriansmares) [inline pkg/applicationserver/io/packages/redis/registry.go:428]: Same here.

File: pkg/applicationserver/io/pubsub/redis/registry.go [hunk 4]
```diff
@@ -214,8 +214,7 @@ func (r PubSubRegistry) Set(ctx context.Context, ids *ttnpb.ApplicationPubSubIde
 				pb = &ttnpb.ApplicationPubSub{}
 			}
 
-			now := time.Now().UTC()
-			pb.UpdatedAt = &now
+			pb.UpdatedAt = ttnpb.ProtoTimePtr(time.Now())
```
Comment (adriansmares) [inline pkg/applicationserver/io/pubsub/redis/registry.go:217]: Same here.

File: pkg/applicationserver/io/web/redis/registry.go [hunk 5]
```diff
@@ -174,8 +174,7 @@ func (r WebhookRegistry) Set(ctx context.Context, ids *ttnpb.ApplicationWebhookI
 				pb = &ttnpb.ApplicationWebhook{}
 			}
 
-			now := time.Now().UTC()
-			pb.UpdatedAt = &now
+			pb.UpdatedAt = ttnpb.ProtoTimePtr(time.Now())
```
Comment (adriansmares) [inline pkg/applicationserver/io/web/redis/registry.go:177]: Same here.

File: pkg/applicationserver/io/packages/redis/registry.go (hunk 2 above)
Comment (htdvisser) [reply pkg/applicationserver/io/packages/redis/registry.go:233]: Proto time is just Unix time with nanosecond precision. It doesn't carry timezone info, so `UTC()` jost isn't necessary. I've also removed it from the places where I didn't remove it yet.

**Review by adriansmares — APPROVED** [review]

**Pull request merged**